pkgs/applications/graphics/round/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "round";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "mingrammer";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "09brjr3h4qnhlidxlki1by5anahxy16ai078zm4k7ryl579amzdw";
+  };
+
+  #vendorSha256 = "17v5nzswaaaabawxrvkza8a5d9aqyhb2gkncp3fqv6rszmkwwww6";  # there isn't any?
```

Comment:
Is that a question or a statement (both here, below this line and in the other file)?

If there aren't any dependencies why aren't you removing the commented out line with the vendorSha256?

Reply:
It was a statement. Sorry, I PR'ed this is a messy state. Cleaning